README.md

```diff
@@ -40,6 +40,7 @@ Welcome to the [Next Generation's Team Workshop Library](presentation.pptx), bui
 1️⃣ | [Build a Website to Showcase your Resume](./full/build-resume-website/README.md) | Ornella Altunyan | 1 hour  | Basics of HTML and CSS to showcase your skills in an online resume | [🎥](https://youtu.be/M2IrPFMFwx8) | HTML, CSS, GitHub Pages, [Web development](https://docs.microsoft.com/learn/paths/web-development-101/?WT.mc_id=academic-51109-ornella) |
 1️⃣ | [Build an App without Using any Code with Microsoft Power Platform](./full/power-platform-canvas-app/README.md) | Haimantika Mitra | 1 hour  | Build a no-code App using Power Platform | | Power Apps, Power Platform, [Canvas apps](https://docs.microsoft.com/learn/modules/build-app-solution/?WT.mc_id=academic-56577-hmitra) |
 1️⃣ | [Getting Started with Development Containers in VS Code](./full/using-dev-containers-vscode/README.md) | Ornella Altunyan | 1 hour | How to leverage development containers in Visual Studio Code to collaborate across machines and development environments. | | Git, [Use a Docker container as a development environment with Visual Studio Code](https://docs.microsoft.com/learn/modules/use-docker-container-dev-env-vs-code/?WT.mc_id=academic-55190-ornella) |
+1️⃣ | [Build a Minimal API with .NET 6](./full/intro-minapi/README.md) | Katie Savage | 1 hour | Build your first web API by building a Todo list. | | .NET 6, SQLite, [Build a web API with minimal API, ASP.NET Core, and .NET 6](https://docs.microsoft.com/learn/modules/build-web-api-minimal-api/) |
```

Comment:
I'm wondering whether this is a level 2, what do you think?

Reply:
Agreed. I'll switch it to 2.